src/app/run/pipelines/pipelines-routing.module.ts

```diff
@@ -1,17 +1,19 @@
-import { NgModule }  from '@angular/core';
+import { AuthGuard } from './../../shared/auth-guard.service';
```

Comment:
For consistency we have the imports done in the groups. First is Angular, then external modules, then internal. 

Sometimes I try to alphabetize them within their group too.